src/assets/styles/collections/_projects.css

```diff
@@ -130,9 +130,9 @@
 @media (width >= 70.875rem) {
     .project .banner {
         display: grid;
-        gap: 0;
+        gap: var(--common-inline-padding);
```

Comment:
This causes a problem because now we have a calculated width which = 35% + 65% + --common-inline-padding, which adds up to more than 100%. So the banner title overflows at some screen sizes.